sycl/include/sycl/ext/intel/experimental/fpga_mem/properties.hpp

```diff
@@ -0,0 +1,323 @@
+//==----- properties.hpp - SYCL properties associated with fpga_mem ---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <sycl/ext/oneapi/properties/property.hpp>       // for PropKind
+#include <sycl/ext/oneapi/properties/property_value.hpp> // for property_value
+
+#include <cstdint>     // for uint16_t
+#include <iosfwd>      // for nullptr_t
+#include <string_view> // for string_view
+#include <type_traits> // for true_type
+
+namespace sycl {
+inline namespace _V1 {
+namespace ext {
+namespace intel {
+namespace experimental {
```

Comment:
Nit; We can merge these into one now, so we might as well.
```suggestion
namespace ext::intel::experimental {
```

Reply:
`ext` can't be folded into a single line, but `intel::experimental` can